storage/Filesystems/MountPoint.cc

```diff
@@ -137,6 +137,13 @@ namespace storage
     }
 
 
+    vector<string>
+    MountPoint::default_mount_options() const
+    {
+	return get_impl().default_mount_options().get_opts();
```

Comment:
NP: indentation looks wrong, doesn't it?

Reply:
I think it's right, consistent to the rest of the file.

Reply:
Ok, github is not rendering it as I expected.